src/test/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/OuderOfPartnerOverledenOfOnbevoegdTotGezagTest.java

```diff
@@ -97,7 +98,7 @@ void ouderOfPartnerOverledenOfOnbevoegdTotGezagWithOuderOverledenOfOnbevoegd() {
     void ouderOfPartnerOverledenOfOnbevoegdTotGezagWithNietOuderOverledenOfOnbevoegd() {
         when(gezagsBepaling.getArAntwoordenModel()).thenReturn(arAntwoordenModel);
         when(persoonslijstOuder1.isOverledenOfOnbevoegd()).thenReturn(false);
-        when(persoonslijstNietOuder.isOverledenOfOnbevoegd()).thenReturn(true);
+        when(persoonslijstNietOuder.isOverledenOfOnbevoegdEncoded()).thenReturn(Optional.of('o'));
```

Comment:
nitpick: consisten zou zijn om de 'o' als constante te definieren met een beschrijvende naam

Reply:
`o` is nu een constante.